packages/salesforcedx-vscode-soql/src/editor/soqlEditorInstance.ts

```diff
@@ -181,11 +182,23 @@ export class SOQLEditorInstance {
         break;
       }
       case MessageType.RUN_SOQL_QUERY: {
-        this.handleRunQuery().catch(() => {
-          channelService.appendLine(
-            `An error occurred while running the SOQL query.`
-          );
-        });
+        vscode.window
+          .withProgress(
+            {
+              cancellable: false,
+              location: vscode.ProgressLocation.Notification,
+              title: 'Running query'
+            },
+            () => {
+              return this.handleRunQuery();
+            }
+          )
+          .then(undefined, () => {
+            channelService.appendLine(
+              `An error occurred while running the SOQL query.`
```

Comment:
Same, let's add this to `src/messages/i18n.ts`.
cc @ekapner

Reply:
I've got these error messages defined in i18n with another PR.

Reply:
@jgellin-sf @dehru let's not keep doing this thing of breaking up work so that we have follow up PRs exclusively for cleaning up previously unfinished work. Let's update this PR to use the labels it needs in the pattern we already have (i18n.ts file).